server/controllers/requestController.js

```diff
@@ -0,0 +1,32 @@
+import requestsDb from '../dummyModel/request';
+/**
+ * class Request controls all request methods
+ * @class request
+ */
+export default class Requests {
+	/**
+	 * adds a new request
+	 * @param {object} req
+	 * @param {object} res
+	 * @returns {json} adds new request
+	 * @memberof Request
+	 */
+	static addRequest(req, res) {
+    const { title, department, equipment,serialNumber,description } = req.body;
+    const id = requestsDb.length;
+    const request = {
+      title,
+      department,
+      equipment,
+      serialNumber,
+      description
+    };
+    requestsDb.push(request);
+    return res.status(201)
+      .json({
```

Comment:
Use the JSend response format as shown here: https://sergeyakopov.com/integrating-jsend-response-format-with-your-web-api/

Reply:
okay